pyopencl/cache.py

```diff
@@ -98,8 +98,8 @@ def __init__(self, cleanup_m, cache_dir):
                     warn("could not obtain cache lock--delete '%s' if necessary"
```

Comment:
This logic no longer makes sense. It'll now print this every 0.05 seconds.

Reply:
Hmm, I'm not sure what to do here... Remove the warning completely, or print it just for `attempts == 20`?

Reply:
It should be printed at reasonable intervals, every ten seconds or so, IMO.

Reply:
Done in 8ab383a